synapseclient/__main__.py

```diff
@@ -1871,8 +1871,10 @@ def _authenticate_login(
 
 def main():
     args = build_parser().parse_args()
-    synapseclient.USER_AGENT["User-Agent"] = (
-        "synapsecommandlineclient " + synapseclient.USER_AGENT["User-Agent"]
+    synapseclient.USER_AGENT[
+        "User-Agent"
+    ] = "synapsecommandlineclient" + synapseclient.USER_AGENT["User-Agent"].replace(
```

Comment:
Nice clever trick! Could we just get rid of the + and replace synapseclient with synapsecommandlineclient?